src/common/containers/ZeroApprovalModal/ZeroApprovalModal.tsx

```diff
@@ -35,15 +32,14 @@ export function ZeroApprovalModal({ onDismiss = () => {} }: ZeroApprovalModalPro
     }
   }, [isApproving, hasUserClosedModal])
 
-  const { walletName, ensName } = walletDetails
-  const walletAddress = ensName || (account ? shortenAddress(account) : '')
+  const { walletName } = walletDetails
```

Comment:
~No longer using ENS name?~
~Or has ENS name been integrated onto walletDetails?~

Got to the hook and you are doing exactly that 👍